doc/recipe/preprocessor.rst

```diff
@@ -59,7 +59,8 @@ Overview
 The ESMValTool preprocessor can be used to perform a broad range of operations
 on the input data before diagnostics or metrics are applied. The preprocessor
 performs these operations in a centralized, documented and efficient way, thus
-reducing the data processing load on the diagnostics side.
+reducing the data processing load on the diagnostics side.  For an overview of
+the preprocessor structure see the `recipe overview <https://docs.esmvaltool.org/projects/esmvalcore/en/latest/recipe/overview.html#recipe-section-preprocessors>`_.
```

Comment:
```suggestion
the preprocessor structure see the :ref:`Preprocessors`.
```
It would be better to use the label to create the link to the other section, that will work with multiple versions of the documentation.